azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java

```diff
@@ -118,6 +118,9 @@ public static ExecutionOptions parseFlowOptions(final HttpServletRequest req,
     final Map<String, String> flowParamGroup = getParamGroup(req, "flowOverride");
     execOptions.addAllFlowParameters(flowParamGroup);
 
+    final Map<String, Map<String, String>> jobParamGroup = getMapParamGroup(req, "jobOverride");
```

Comment:
Since users will be able to specify both job and nested flow properties with this new parameter, what do you think about naming it `nodeOverride` which is more generic?

Reply:
Indeed. So shall I rename it?

Reply:
yes please, `nodeRuntimeProperties` could be another alternative but I guess `nodeOverride` has more similarity with existing `flowOverride` and might be easier to understand, at least for folks already familiar with the APIs. I'm fine either way.

Reply:
Renaming jobX -> nodeX done.